### DISCUSSION TITLE
How to serve locally? vite?

### DISCUSSION THREAD

**Discussion by mdrie** [StaticJsCMS/static-cms#99] [Q&A]
I use vite to serve a bundled Netlify CMS locally.

With Static CMS I get
```
 [ERROR] Big integer literals are not available in the configured\
 target environment ("chrome87", "edge88", "es2020", "firefox78", "safari13" + 2 overrides)
```
It seems, you built it quite differently. Should I file an issue for this or is there a better way?

I use `elm-pages` as front-end, whose dev server is currently not capable of serving the CMS. Therefore, I am quite flexible and can easily use a different dev server for the CMS.

**Comment by KaneFreeman** [comment]
Yes please open an issue. Might need an example of your repo's setup to figure out the issue. The build process for the CMS has changed quite a bit (and will probably change more in the future).
